src/main/webapp/index.html

```diff
@@ -44,6 +43,7 @@
                     <textarea id="search-course" maxlength=1500 placeholder="Class Name"></textarea>
                     <textarea id="search-prof" maxlength=1500 placeholder="Professor"></textarea>
                 </div>
+                <button onclick=verify()>Verify</button>
```

Comment:
Do you need this here?

Reply:
Oops no, was just using for testing. Will remove!